charts/plex/templates/deployment.yaml

```diff
@@ -69,7 +69,32 @@ spec:
           imagePullPolicy: {{ .Values.image.pullPolicy }}
           ports:
           - name: pms
+            protocol: TCP
             containerPort: 32400
+          - name: http
+            protocol: TCP
+            containerPort: 32400
+          - name: https
+            protocol: TCP
+            containerPort: 32443
```

Comment:
In #212 the http & https named ports were removed because,

>  In the recent releases http and https all use 32400, so this release only uses the single port for the service, and only 32400 is exposed in the container.

Do you think we should add these back or keep them removed?

Reply:
Let's get em outta here